From: SeongJae Park <sj@kernel.org>
Date: Sun, 13 Jul 2025 13:47:24 -0700
Subject: [PATCH] ==== more selftest ====

selftests/damon: test all parameters

sysfs.py is testing if only a subset of DAMON parameters are being
passed to DAMON in kernel space, as expected.  Extend the test coverage
to all parameters.

First X patches extend _damon_sysfs.py to support specifying and
committing all DAMON parameters.   X patches extending the drgn script
to dump whole DAMON parameters follow.  Finally, following X patches
update sysfs.py to set the whole parameters via sysfs, and ensure the
parameters are passed to DAMON in kernel space using the drgn-based
DAMON status dumping script.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_75q62vqb | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_75q62vqb

diff --git a/hkml_cv_bogus/hkml_cv_bogus_75q62vqb b/hkml_cv_bogus/hkml_cv_bogus_75q62vqb
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

